host/src/PlaceholderApp/FunctionsNetHost.PlaceholderApp.sln

```diff
@@ -0,0 +1,31 @@
+﻿
```

Comment:
nit: remove extra line

Reply:
This was generated by the tool. So, I am leaving it as it is.